content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Mosaic visualization
+list_title: Mosaic
+list_image: /img/influxdb/2-0-visualizations-mosaic-example.png
+description: >
+  The Mosaic view displays the single value most recent value for a time series in a mosaic view.
+weight: 206
+menu:
+  influxdb_2_0:
+    name: Mosaic
+    parent: Visualization types
+---
+
+The **Mosaic** visualization displays data from two or more qualitative variables.
+
+{{< img-hd src="/img/influxdb/2-0-visualizations-mosaic-example.png" alt="mosaic example" />}}
+
+Select the **Mosaic** option from the visualization dropdown in the upper left.
+
+## Mosaic behavior
+The mosaic visualization displays string values in a graph.
```

Comment:
Not sure i understand this line or section. The mosaic allows you to view state changes in your time series data visually.